### PR TITLE
Feature/feedback thank you

### PR DESCRIPTION
**Pull request by omarzoghayyer**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_
Solves FEEDBACK- The 'Thank you!' message needs to be more appropriate #1126
Screen Shot 2020-07-11 at 9 09 53 PM

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] This branch is rebased/merged with the **latest master**.
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/src/templates/layouts/NavigationLayout.js [hunk 1]
```diff
@@ -310,7 +310,6 @@ const NavigationLayout = (props) => {
       });
     }
   };
-
   const renderThanksModal = () => (
```
Comment (nr431207) [inline client/src/templates/layouts/NavigationLayout.js:313]: add the line space back, between 312 and 313.

File: client/src/templates/layouts/NavigationLayout.js [hunk 2]
```diff
@@ -320,10 +319,12 @@ const NavigationLayout = (props) => {
...
       <h2 className="title">Thank you!</h2>
       <p>
-        Your input means a lot and helps us help you and others during and after
-        the COVID-19 pandemic.
+        Your input means a lot and helps us improve our services during and
+        after the COVID-19 pandemic.
       </p>
-      <Logo src={logo} alt="FightPandemics logo" />
+      <Link to="/">
+        <Logo src={logo} alt="Fight Pandemics logo" />
```
Comment (idjevm) [inline client/src/templates/layouts/NavigationLayout.js:326]: Please change back to `FightPandemics`

File: client/src/templates/layouts/NavigationLayout.js (hunk 1 above)
Comment (omarzoghayyer) [reply client/src/templates/layouts/NavigationLayout.js]: Thats odd! Auto prettier deleted it but It makes sense to have it. Thank you for pointing this out.

**Comment by idjevm** [comment]
Ohh wait, revisiting this again, now that I think about it, based on the threads, conversations, I think @mannykary mentioned that right now we have staging being what's on the master branch, so that's frozen for right now, so this code (the one in rc/v1.1) is not deployed after merged, and because of that, testing should happen on the PR Build URL. So we will be tagging the QA team around the PRs so @rabbi08, @galeza, @sheeba1, @nivetheatha, other QA can test them on the Build URL rather than on staging.

**Pull request merged**